packages/visx-demo/yarn.lock

```diff
@@ -0,0 +1,7675 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
since `visx` uses yarn workspaces, we have only committed the `yarn.lock` file in the repo root directory. so I think if you remove this file we can just rely on the updated root `yarn.lock` you've updated.

Reply:
Yes, definitely! I didn't notice that I added this file accidentally.